### PR TITLE
Remove back arrow from course management

### PR DESCRIPTION
**Pull request by ivanchimeno**
### Checklist
- [x] I tested *all* changes and *all* related features with different users (student, tutor, instructor, admin) on the test server https://artemistest.ase.in.tum.de.
- [x] Client: I followed the [coding and design guidelines](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/client.html).
- [x] Client: I added multiple screenshots/screencasts of my UI changes

### Motivation and Context

Breadcrumb navigation is used in all `Course Management` and `Server Administration` pages. This replaces the "back arrow" so we remove those from all pages.

### Description

Remove the back arrows from all sub-pages.

### Steps for Testing

1. Log in to Artemis
2. Navigate to Course Administration
3. Select Assessment Board
4. Back arrow should be gone in that view
4. Click on Complaints about your assesement
5. Back arrow should be gone in that view
6. Go back and click on Assesements locked by you
7. Back arrow should be gone in that view

### Test Coverage

Doesn't change since only HTML templates are affected. 
### Screenshots

Screenshot from 2020-11-22 11-52-43
Screenshot from 2020-11-22 11-53-02
Screenshot from 2020-11-22 11-53-34

### CODE REVIEW

**Review by FlorianStang — APPROVED** [review]
Tested on TS1, everything worked as intended.

**Review by JohannesWeiss — APPROVED** [review]
Works as intended.

**Review by airelawaleria — APPROVED** [review]
Works fine

**Review by marvinchimeka — APPROVED** [review]
Worked as described.

**Review by EneaGore — DISMISSED** [review]
Everything fine as described.

**Review by JannisBittlmayer — DISMISSED** [review]
Works as intended.

File: src/main/webapp/app/course/dashboards/assessment-dashboard/assessment-dashboard.component.html [hunk 1]
```diff
@@ -2,7 +2,6 @@
     <div class="row justify-content-between">
         <div class="col-8">
             <h2>
-                <fa-icon [icon]="'arrow-left'" (click)="back()" class="back-button mr-2"></fa-icon>
```
Comment (jpbernius) [inline src/main/webapp/app/course/dashboards/assessment-dashboard/assessment-dashboard.component.html:5]: The `back()` method in the `.component.ts` file is now unused.

File: src/main/webapp/app/complaints/list-of-complaints/list-of-complaints.component.html [hunk 2]
```diff
@@ -2,7 +2,6 @@
     <div class="row justify-content-between">
         <div class="col-8">
             <h2>
-                <fa-icon [icon]="'arrow-left'" (click)="back()" class="back-button mr-2"></fa-icon>
```
Comment (jpbernius) [inline src/main/webapp/app/complaints/list-of-complaints/list-of-complaints.component.html:5]: The `back()` method in the `.component.ts` file is now unused.

File: src/main/webapp/app/assessment/assessment-locks/assessment-locks.component.html [hunk 3]
```diff
@@ -2,7 +2,6 @@
     <div class="row justify-content-between">
         <div class="col-8">
             <h2>
-                <fa-icon [icon]="'arrow-left'" (click)="back()" class="back-button mr-2"></fa-icon>
```
Comment (jpbernius) [inline src/main/webapp/app/assessment/assessment-locks/assessment-locks.component.html:5]: The `back()` method in the `.component.ts` file is now unused.

**Review by jpbernius — CHANGES_REQUESTED** [review]
Please also remove the methods that were called from the Back Arrow Icons.

File: src/main/webapp/app/course/dashboards/assessment-dashboard/assessment-dashboard.component.html (hunk 1 above)
Comment (ivanchimeno) [reply src/main/webapp/app/course/dashboards/assessment-dashboard/assessment-dashboard.component.html:5]: Whoops, thanks for pointing that out!

**Review by DenizShabani — APPROVED** [review]
Everything worked fine for me.

**Review by airelawaleria — APPROVED** [review]
Tested again. Still works, re-approve

**Review by JohannesWeiss — APPROVED** [review]
Still works as intended, re-approve.

**Review by jpbernius — APPROVED** [review]
Let's get this merged! :shipit:

**Pull request merged**